frontend/model/state.js

```diff
@@ -56,6 +56,11 @@ const initialState = {
     : ['error', 'warn', 'info']
 }
 
+const reactiveDate = Vue.observable({ date: new Date() })
+// every 5 minutes update the reactive date to force recomputation of getters using this
+// TODO: can make this every minute after payments pagination no longer loads all of the payments
+setInterval(function () { reactiveDate.date = new Date() }, 5 * 60 * 1000)
```

Comment:
I just realized there's a far better way of writing this, could you please update this to this:

```js
const reactiveDate = Vue.observable({ date: new Date() })
setInterval(function () {
  const date = new Date()
  // payments recalculation happen within a minute of day switchover
  if (Math.abs(reactiveDate.date.getTime() - date.getTime()) >= MINS_MILLIS) {
    reactiveDate.date = date
  }
}, 60 * 1000)
```

Note: you'll need to import `MINS_MILLIS` from `time.js`

Reply:
Got it.